lib/active_merchant/billing/gateways/komoju.rb

```diff
@@ -89,6 +84,25 @@ def commit(params)
                      :authorization => success ? response["id"] : nil)
       end
 
+      def post_data(params)
```

Comment:
Could we use a JSON string as the post body instead? Not sure if we need to convert a hash to a querystring, easier to use `to_json` for this, no?